README_CONTAINER_IMAGE.md

```diff
@@ -0,0 +1,38 @@
+# Containerized openshift-ansible to run playbooks
+
+The [Dockerfile](Dockerfile) in this repository uses the [playbook2image](https://github.com/aweiteka/playbook2image) source-to-image base image to containerize `openshift-ansible`. The resulting image can run any of the provided playbooks.
+
+## Build
+
+To build a container image of `openshift-ansible`:
+
+1. Using standalone **Docker**:
+
+        cd openshift-ansible
+        docker build -t openshift-ansible .
+
+1. Using an **OpenShift** build:
+
+        oc new-build docker.io/aweiteka/playbook2image~https://github.com/openshift/openshift-ansible
+        oc describe imagestream openshift-ansible
+
+## Usage
+
+The base image provides several options to control the behaviour of the containers. For more details on these options see the [playbook2image](https://github.com/aweiteka/playbook2image) documentation.
+
+At the very least, when running a container using an image built this way you must specify:
+
+1. The **playbook** to run. This is set using the `PLAYBOOK_FILE` environment variable.
+1. An **inventory** file. One way to provide this is to serve the inventory file from a web server and use the `INVENTORY_URL` environment variable to fetch it.
```

Comment:
I'd suggest to document the use of INVENTORY_FILE instead.

Users will already need to learn/deal with mounts for SSH keys, so I think using that for the inventory would be more natural than have people figure out how to serve their file over HTTP.